rds.tf

```diff
@@ -138,3 +138,99 @@ resource "aws_db_parameter_group" "mysql5-7-etleap" {
     apply_method = "pending-reboot"
   }
 }
+
+resource "aws_db_parameter_group" "mysql5-6-etleap" {
```

Comment:
Do we need to keep this around?

Reply:
Yes at least for one more version. If we delete this in this version there will be a hard dependency: terraform will try to delete mysql5.6-parameter-group before the upgrade, thus blocking the upgrade path.
After the upgrade - next version - we can safely remove this unused parameter group.